packages/backend-app-api/src/logging/WinstonLogger.ts

```diff
@@ -85,24 +86,17 @@ export class WinstonLogger implements RootLoggerService {
 
     let redactionPattern: RegExp | undefined = undefined;
 
-    const replace = (obj: TransformableInfo) => {
-      for (const key in obj) {
-        if (Object.hasOwn(obj, key)) {
-          if (typeof obj[key] === 'object') {
-            obj[key] = replace(obj[key] as TransformableInfo);
-          } else if (typeof obj[key] === 'string') {
-            try {
-              obj[key] = obj[key]?.replace(redactionPattern, '[REDACTED]');
-            } catch {
-              /* ignore read only properties */
-            }
-          }
-        }
-      }
-      return obj;
-    };
     return {
-      format: format(replace)(),
+      format: format((obj: TransformableInfo) => {
+        if (!redactionPattern || !obj) {
+          return obj;
+        }
+
+        obj[MESSAGE] = obj[MESSAGE]?.replace?.(redactionPattern, '[REDACTED]');
+        obj.message = obj.message.replace?.(redactionPattern, '[REDACTED]');
```

Comment:
I just did this for backwards compatibility if anyone has written anything that doesn't use the `MESSAGE` symbol, but maybe we could actually remove it @Rugvip?

Reply:
Yeah I think it should be safe to skip this tbh